Makefile.maven

```diff
@@ -24,4 +24,8 @@ java_install:
 .PHONY: java_clean
 java_clean:
 	echo "Cleaning Maven build ..."
-	mvn clean
\ No newline at end of file
+	mvn clean
+
+.PHONY: spotbugs
+spotbugs: java_compile
```

Comment:
Do you really want to run `java_compile` before it? That seems like it would cause a lot of time waste in the CI.

Reply:
But spotbugs needs the new code compiled in order to spot ... bugs :-)
This is what we need when developing locally, or?

Reply:
But it means you will recompile everything in CIs if you do it like this. We do not do that for example in operators.

Reply:
Agree. It is useful for local development when you could forget to recompile while fixing spotbugs :-) But doesn't make sense in CI. I removed it.